src/style.css

```diff
@@ -0,0 +1,95 @@
+body {
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+}
+
+.block {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  flex-grow: 1;
+  height: 300px;
+  width: 300px;
+  color: white;
+  background-color: rgb(255, 0, 0);
+  font-family: Arial, sans-serif;
+  font-size: 100px;
+  padding: 0;
+  margin: 0;
+}
+
+.block-2 {
+  background-color: rgb(204, 0, 0);
+}
+
+.block-3 {
+  background-color: rgb(153, 0, 0);
+}
+
+.block-4 {
+  background-color: rgb(102, 0, 0);
+}
+
+.block-5 {
+  background-color: rgb(51, 0, 0);
+}
+
+.block-6 {
+  background-color: rgb(0, 0, 0);
+}
+
+@media (min-width: 450px) {
+  .container {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+  }
+
+  .block-3 {
+    order: 4;
+  }
+
+  .block-4 {
+    order: 3;
+  }
+
+  .block-5 {
+    order: 5;
+  }
+
+  .block-6 {
+    order: 6;
+  }
+}
+
+@media (min-width: 900px) {
+  .container {
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+  }
+
+  .block {
+    flex-basis: 33.33%;
+  }
```

Comment:
33.33*3 is not quite 100

Reply:
Yes it's 99.99999999999...... Am trying to find a way to do it :)